tracery/modifiers.py

```diff
@@ -67,3 +67,130 @@ def lowercase(text, *params):
     'uppercase': uppercase,
     'lowercase': lowercase
 }
+
+
+#----
+#inflect.py modifiers
+#Mappings based on https://github.com/pwdyson/inflect.py
+
+import inflect
+p = inflect.engine()
+
+
+def ordinal(text, *params):
+    return p.ordinal(text)
+
+def number_to_words(text, *params):
+    kwargs=dict(param.split('=') for param in params)
+    try:
+        resp = p.number_to_words(text, **kwargs)
+    except:
+        resp = "no recorded"
+    return resp
+    
+
+def plural(text, *params):
+    if params: return p.plural(text, params[0])
+    return p.plural(text)
+
+def plural_noun(text, *params):
+    if params: return p.plural_noun(text, params[0])
+    return p.plural_noun(text)
+
+def plural_verb(text, *params):
+    if params: return p.plural_verb(text, params[0])
+    return p.plural_verb(text)
+
+def plural_adj(text, *params):
+    if len(params): return p.plural_adj(text, params[0])
+    return p.plural_adj(text)
+    
+def singular_noun(text, *params):
+    return p.singular_noun(text)
+    
+def no(text, *params):
+    return p.no(text)
+    
+def num(text, *params):
+    return p.num(text)
+    
+def compare(text, *params):
+    return p.compare(text, params[0])
+    
+def compare_nouns(text, *params):
+    return p.compare_nouns(text,params[0])
+
+def compare_adjs(text, *params):
+    return p.compare_adjs(text,params[0])
+
+def a(text, *params):
```

Comment:
this function definition is overwriting the previous definition of a function with the same name above (which is why the test is failing)